hs_collection_resource/patches/add_hasPart_term_django_shell.py

```diff
@@ -0,0 +1,57 @@
+# This script is to add "hasPart" metadata to all collection resources (github issue #1403)
+# how to run:
+# docker exec -i hydroshare python manage.py shell \
+# < "hs_collection_resource/patches/add_hasPart_term_django_shell.py"
+# Note: use "-i" instead of "-it" in above command as
+# the latter may cause error "cannot enable tty mode on non tty input"
+
+from datetime import datetime
```

Comment:
@pkdash 
Hong helped run this script on riverdev.  Jeff and I did some manual checks on affected resources and they looked good so far.